java/codec-template.java.j2

```diff
@@ -196,28 +229,43 @@ public final class {{ service_name|capital }}{{ method.name|capital }}Codec {
     {% endfor %}
         return request;
     }
+{% endif %}
 
 {#RESPONSE PARAMETERS#}
+{% set noResponseValue = method.response.params|length == 0 and response_new_params|length == 0 %}
+{% set singleResponseValue = method.response.params|length == 1 and response_new_params|length == 0 %}
+{% if noResponseValue %}
+{% elif singleResponseValue %}
+    {% set param = method.response.params|last %}
+    /**
+    {% for line in param.doc.splitlines() %}
+     * {{ line }}
+    {% endfor %}
+     */
+    @edu.umd.cs.findbugs.annotations.SuppressFBWarnings({"UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD"})
+    public {% if param.nullable  %}@Nullable {% endif %}{{ lang_types_decode(param.type) }} {{ param.name }};
+{% else %}
     @edu.umd.cs.findbugs.annotations.SuppressFBWarnings({"URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"})
     public static class ResponseParameters {
-{% for param in method.response.params %}
+    {% for param in method.response.params %}
+            /**
```

Comment:
Can you decrease the indent level by one for the javadoc and fields ? Right now, RequestParameters has 4 spaces for their fields whereas ResponseParams has  8.

Reply:
I could not find where the problem is. Generated code also looks correct. 
Request and Response Parameters have 8 spaces if they are multivalue(because they inside another class). 
When they are single value both have 4 spaces.

Reply:
Ok I saw the problem. 👍